packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -63,14 +65,18 @@ import {NoWork, Never} from './ReactFiberExpirationTime';
 import {AsyncMode, StrictMode} from './ReactTypeOfMode';
 import MAX_SIGNED_31_BIT_INT from './maxSigned31BitInt';
 
+const {getCurrentFiberStackAddendum} = ReactDebugCurrentFiber;
+
 let didWarnAboutBadClass;
 let didWarnAboutGetDerivedStateOnFunctionalComponent;
 let didWarnAboutStatelessRefs;
+let getStack = emptyFunction.thatReturns('');
```

Comment:
`getStack` would only ever be called in DEV, so we can pass `getCurrentFiberStackAddendum` to `checkPropTypes` directly and avoid the `getStack` variable altogether.